ynr/apps/uk_results/management/commands/uk_results_mark_uncontested.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+from candidates.models import Ballot
+
+
+class Command(BaseCommand):
+
+    help = """
+    Find and mark any uncontested ballots not already marked after ballot lock
+    """
+
+    def handle(self, *args):
+        qs = Ballot.objects.all()
+        for ballot in qs:
+            try:
```

Comment:
Added a related comment above - but when using a `try/except` it is good practice attempt to catch a specific exception that you know might happen that you want to catch (e.g. when iterating though a list you might sometimes expect `IndexError` and want to catch that). Rather than catch any exception. But here you don't need one, as your logic in `uncontested` should have dealt with everything.

Also, I think `ballot.mark_person_as_elected()` is wrong isn't it?

Reply:
Thanks for the explanation. 

Yes, very wrong :) Thanks for catching that.